packages/date-time-picker/package.json

```diff
@@ -1,18 +1,18 @@
 {
-    "name": "@spectrum-web-components/time-field",
+    "name": "@spectrum-web-components/date-time-picker",
```

Comment:
Confirming this is meant to rename the other working package, not create a new package in parallel.